App.js

```diff
@@ -10,19 +9,28 @@ import DatePicker from './components/DatePicker/DatePicker';
 import Dashboard from './containers/Dashboard/Dashboard'
 import { StyleSheet, View, Image, Text, ScrollView } from 'react-native';
 
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import { rootReducer } from './reducers/index';
+import { connect } from 'react-redux';
```

Comment:
very smart getting redux set up here, should definitely help make things a lot easier